velox/exec/HashPartitionFunction.cpp

```diff
@@ -57,7 +59,12 @@ void HashPartitionFunction::init(
 std::optional<uint32_t> HashPartitionFunction::partition(
     const RowVector& input,
     std::vector<uint32_t>& partitions) {
-  auto size = input.size();
+  const auto size = input.size();
+
+  if (hashers_.empty()) {
+    partitions.resize(size, 0u);
```

Comment:
Please, remove the code to fill 'partitions' with zeros.

Reply:
Yeah just wanted to remove this after https://github.com/facebookincubator/velox/pull/6534